### PR TITLE
Storage rework

### PR DESCRIPTION
**Pull request by gctucker**
Rework the monitor, trigger and build jobs to use the new Storage configuration.

### CODE REVIEW

**Comment by nuclearcat** [comment]
It looks like kernelci-core doesnt support --storage-config yet?
```
./kci_build push_tarball --storage-config
usage: kci_build [-h] [--yaml-config YAML_CONFIG] [--settings SETTINGS]
                 {validate,list_configs,check_new_commit,update_last_commit,tree_branch,update_mirror,update_repo,describe,generate_fragments,generate_defconfig_fragments,expand_fragments,push_tarball,list_variants,arch_list,get_build_env,get_reference,show_build_env,list_kernel_configs,init_bmeta,make_config,fetch_firmware,make_kernel,make_modules,make_dtbs,make_kselftest,push_kernel,pull_tarball}
                 ...
kci_build: error: unrecognized arguments: --storage-config
```

**Comment by gctucker** [comment]
Yes I'm making the PR right now.

**Comment by gctucker** [comment]
Tested OK on staging with my own branch and Docker images:
https://bot.staging.kernelci.org/job/gtucker/job/kernel-build-trigger/28/console

This is now being tested on staging with the regular integration branch.

File: jobs/monitor.jpl [hunk 1]
```diff
@@ -65,7 +65,7 @@ def checkConfig(config) {
 kci_build \
 check_new_commit \
 --build-config=${config} \
---storage=${params.KCI_STORAGE_URL} \
+--storage-config=${params.KCI_STORAGE_CONFIG} \
```
Comment (gctucker) [inline jobs/monitor.jpl:68]: Tested OK on staging:
https://bot.staging.kernelci.org/job/kernel-tree-monitor/1354/console
```
kci_build check_new_commit --build-config=kernelci_staging-next --storage-config=staging.kernelci.org
```

File: jobs/rootfs-builder.jpl [hunk 2]
```diff
@@ -73,8 +73,8 @@ def upload(config, pipeline_version, kci_core, rootfs_type, arch) {
             sh(script: """\
 ./kci_rootfs \
 upload \
---db-token ${API_TOKEN} \
---api ${params.KCI_API_URL} \
+--storage-config ${KCI_STORAGE_CONFIG} \
+--storage-cred ${API_TOKEN} \
```
Comment (gctucker) [inline jobs/rootfs-builder.jpl:77]: Tested OK on staging:
https://bot.staging.kernelci.org/job/rootfs-builder/396/consoleFull
```
./kci_rootfs upload --storage-config staging.kernelci.org --storage-cred **** --rootfs-dir 20221220.0/_install_/bullseye-gst-fluster --upload-path images/rootfs/debian/bullseye-gst-fluster/20221220.0
```

**Comment by gctucker** [comment]
Tested on staging in conjunction with https://github.com/kernelci/kernelci-core/pull/1573

**Comment by gctucker** [comment]
@nuclearcat Would you like to do another review now that everything is in place and it seems to be working as expected on staging?

**Review by nuclearcat — APPROVED** [review]
LGTM, works on staging, doesnt break anything

**Pull request merged**